### ISSUE TITLE
Draft notifications blog post

### ISSUE THREAD

**Issue by alexpandel** [GSA/public-benefits-studio#59]
Skeleton [in here 🔒](https://docs.google.com/document/d/1JklA0gpr7MZc4ToDz0ga4fXnrh5fqOCSLSg_oDuYf6o/edit#heading=h.15t9osuto28); this is dependent on #51

**Comment by alexpandel** [comment]
closing in favor of #98
